### PR TITLE
Add info from kbase on troubleshooting sync

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Incorporates info from https://www.suse.com/support/kb/doc/?id=000018198 to existing tshoot-sync page in Admin Guide.

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [ ] Manager-4.1
- [x] Manager-4.2
- [x] Uyuni

# Links

Fixes https://github.com/uyuni-project/uyuni-docs/issues/515

### CODE REVIEW

File: modules/administration/pages/tshoot-sync.adoc [hunk 1]
```diff
@@ -32,3 +43,52 @@ Please run spacewalk-repo-sync in interactive mode to import it.
...
++
+Another problem that can cause a timeout is an incomplete package download.
+To fix this, check the [path]``/var/spacewalk/packages/1/stage/`` directory and remove any partially downloaded file.
++
+If you use a cron job to run your synchronization automatically, use this code in your cron file to clean the stage directory before you begin:
++
+----
+# connect to customer center every day at random time
+# between 03:03 and 05:50
+3 3 * * * sleep $[ $RANDOM / 5 ]; rm /var/spacewalk/packages/1/stage/* && /usr/sbin/mgr-ncc-sync >/dev/null 2>/dev/null
```
Comment (mcalmer) [inline modules/administration/pages/tshoot-sync.adoc:93]: This command combination is a bad idea.

Every SUMA uses taskomatic to run sync on regular base. Having now cron doing the same is a bad idea as they can conflict. Beside of this, the mgr-ncc-sync is so outdated, i think it does not exist anymore.

Do not suggest a cron job and remove the mgr-ncc-sync command.
When there is a problem, just remove the packages from stage/1 and add a spacewalk-repo-sync example like in the other sections.

File: modules/administration/pages/tshoot-sync.adoc [hunk 2]
```diff
@@ -32,3 +43,52 @@ Please run spacewalk-repo-sync in interactive mode to import it.
...
+----
++
+This option verifies the repository data before the synchronization, rather than relying on locally cached checksums.
+
+
+Timeout::
+If you are synchronizing a {redhat} client with a very busy server, you might see a timeout error like this:
++
+----
+Errno 256 - No more mirrors to try
```
Comment (mcalmer) [inline modules/administration/pages/tshoot-sync.adoc:74]: This is not the timeout error. This is general message yum print when something goes wrong.
Currently i am not sure how to really detect a timeout error.

**Review by keichwa — APPROVED** [review]

**Review by mcalmer — CHANGES_REQUESTED** [review]
There is a general problem with some of these suggestions: 

Some are made for yum as download backend, but we use now zypper.
@meaksh should review as well and tell us which sections are obsolete

File: modules/administration/pages/tshoot-sync.adoc (hunk 1 above)
Comment (Loquacity) [reply modules/administration/pages/tshoot-sync.adoc:93]: Yeah, I had a feeling :wink:  Removed that bit.

File: modules/administration/pages/tshoot-sync.adoc (hunk 2 above)
Comment (Loquacity) [reply modules/administration/pages/tshoot-sync.adoc:74]: Removed the error line.

**Review by jcayouette — APPROVED** [review]

File: modules/administration/pages/tshoot-sync.adoc [hunk 3]
```diff
@@ -32,3 +43,37 @@ Please run spacewalk-repo-sync in interactive mode to import it.
...
+You can increase the timeout value in the [path]``/etc/rhn/spacewalk-repo-sync/yum.conf`` configuration file.
+Add or amend this line with a number of seconds:
++
+----
+[main]
+timeout=<value>
+----
++
+Another problem that can cause a timeout is an incomplete package download.
+To fix this, check the [path]``/var/spacewalk/packages/1/stage/`` directory and remove any partially downloaded file.
```
Comment (meaksh) [inline modules/administration/pages/tshoot-sync.adoc:79]: This entire "Timeout" section is obsolete.

```suggestion

```

File: modules/administration/pages/tshoot-sync.adoc [hunk 4]
```diff
@@ -19,6 +19,17 @@ If more detailed instructions are required, put them in a "Resolving" procedure:
...
+Synchronization can fail for a number of reasons.
+To get more information about a connection problem, run this command:
+
+----
+export URLGRABBER_DEBUG=DEBUG
+spacewalk-repo-sync -c <channelname> <options> > /var/log/spacewalk-repo-sync-$(date +%F-%R).log 2>&1
+----
+
+
+GPG Key Mismatch::
```
Comment (meaksh) [inline modules/administration/pages/tshoot-sync.adoc:32]:
```suggestion
And also check logs from Zypper at [path]``/var/log/zypper.log``

GPG Key Mismatch::
```

**Review by meaksh — COMMENTED** [review]
Added some suggestions related with the move to zypper

**Comment by meaksh** [comment]
I've also created this other PR https://github.com/uyuni-project/uyuni-docs/pull/557 pointing to this current one, to incorporate another change that I was not able to add as suggestion here.

**Comment by Loquacity** [comment]
image

**Review by meaksh — APPROVED** [review]

**Pull request merged**